Documentation.md

```diff
@@ -0,0 +1,109 @@
+# Documentation
+Creating different fields on creating a waifu profile involves filling out different characters in order to get display the information
+
+[Type Field](https://github.com/yourWaifu/is-your-waifu-legal/wiki/Documentation#type-field) 
+
+[English Name Field](https://github.com/yourWaifu/is-your-waifu-legal/wiki/Documentation#english-name-field) 
+
+[Aliases Field](https://github.com/yourWaifu/is-your-waifu-legal/wiki/Documentation#aliases-field) 
+
+[Image Field](https://github.com/yourWaifu/is-your-waifu-legal/wiki/Documentation#image-field) 
+
+[Japanese Name Field](https://github.com/yourWaifu/is-your-waifu-legal/wiki/Documentation#japanese-name-field) 
+
+[Year Field](https://github.com/yourWaifu/is-your-waifu-legal/wiki/Documentation#year-field) 
+
+[Definitely Legal Field](https://github.com/yourWaifu/is-your-waifu-legal/wiki/Documentation#definitely-legal-field) 
+
+[Month Field](https://github.com/yourWaifu/is-your-waifu-legal/wiki/Documentation#month-field) 
+
+[Day of Month Field](https://github.com/yourWaifu/is-your-waifu-legal/wiki/Documentation#day-of-month-field)
+
+[Notes Field](https://github.com/yourWaifu/is-your-waifu-legal/wiki/Documentation#notes-field)
+
+[Sources Field](https://github.com/yourWaifu/is-your-waifu-legal/wiki/Documentation#sources-field)
+
+[Final Result](https://github.com/yourWaifu/is-your-waifu-legal/wiki/Documentation#final-result)
+
+## Type Field
+Information not yet provided
```

Comment:
For now, just keep at 0. This is for future different JSON files that aren't waifu data.